views/result.erb

```diff
@@ -0,0 +1,6 @@
+<h2><%=@game.player.name%> played <%=@game.player_move%></h2>
+<h2>computer played <%=@game.computer_move%></h2>
+<h1><%=@game.winner%></h1>
+<form method='get' action='/play'>
+    <button type=submit>Play again?</button>
+</form>
```

Comment:
All of your page views are super clean and easy to understand, I really like it. Just a note for future about using "h" tags, if you want something to be bigger, it's best to change the font size instead of using a bigger "h" tag. This is because screen readers look for the different tags to understand the page so might get confused. I know we haven't done accessibility yet but thought it's interesting to know :)